packages/xo-server/src/xo-mixins/jobs/index.js

```diff
@@ -266,6 +266,19 @@ export default class Jobs {
     })
 
     const app = this._app
+    const user = await app.getUser(job.userId)
+    const data = {
+      callId: Math.random().toString(36).slice(2),
+      method: 'backupNg.runJob',
+      params: {
+        jobId: job.id,
```

Comment:
`backupNg.runJob` doesn't take `jobId` and `scheduleId` as params, but,  it takes `id` and `schedule`. See https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/api/backup-ng.js#L146-L151